README.md

```diff
@@ -48,10 +49,55 @@ const HomePage = () => {
     i18n.locale('de'); // if dict was already loaded
     i18n.locale('de', DE); // set and load dict at the same time
 
-    // Get current locale
-    i18n.locale(); // de
```

Comment:
Why did you remove it?

Reply:
Sorry, it wasn't removed, I started before you put in those changes. We just need to pull in master